### DISCUSSION TITLE
Why adding Enum required?

### DISCUSSION THREAD

**Discussion by notwhoyouexpected** [spatie/laravel-model-status#125] [General]
Hi, isn't that a breaker for a lot of use cases? for example i added over a admin panel allowed statuses which are saved in the database and call center agents can than assign these statuses to their customers... but with enums i cantt to that anymore...

**Comment by freekmurze** [comment]
I've reverted the changes. The last release should work with strings again.

**Reply by notwhoyouexpected** [reply]
Thanks, will you add this in the future again?

**Reply by freekmurze** [reply]
I've asked the contributor of this functionality, to re-add it while guaranteeing backwards compatibility
